msw/pd.nsi

```diff
@@ -65,8 +101,15 @@ FunctionEnd
 
 Name "${PRODUCT_NAME} ${PRODUCT_VERSION}"
 OutFile "/tmp/pd-${PRODUCT_VERSION}.windows-installer.exe"
-InstallDir "$PROGRAMFILES\Pd"
-InstallDirRegKey HKLM "${PRODUCT_DIR_REGKEY}" ""
+InstallDir "$PROGRAMFILES${ARCHI}\Pd"
+
+;
+; removed as of 0.49 to prevent 32/64 installer clashing. No negative side effects were observed.
+;
+;
+;			InstallDirRegKey HKLM "${PRODUCT_DIR_REGKEY}" ""
```

Comment:
looking up the [NSIS Documentation](http://nsis.sourceforge.net/Reference/InstallDirRegKey), I think the usage of this is to remember the installation directory across re-installs. As such, it seems clear that their are no *immediate* side-effects visible when doing a single installation. The change will only become apparent in consecutive use of the installer.
As such, I guess it should stay in (probably with a explanatory comment why it should stay)

Reply:
If that is present you get the wrong default installation dir for the parallel 2nd installation something like: c:\program files\pd\bin. I tested it.

Reply:
i guess, that's because the `InstallDirRegKey` must be different for 32bit and 64bit installations